### PR TITLE
Changes of List view default background color and separator color on windows

### PR DESCRIPTION
**Pull request by Sofiya-kumar**
Default background color of list view

### CODE REVIEW

File: Trimble.Modus.Components/Controls/ListView/TextCell.xaml.cs [hunk 1]
```diff
@@ -53,16 +52,26 @@ public string Description
     public TextCell()
     {
         InitializeComponent();
+        if (DeviceInfo.Platform != DevicePlatform.WinUI)
+        {
+            UpdateBackgroundColor();
```
Comment (rthanga1) [inline Trimble.Modus.Components/Controls/ListView/TextCell.xaml.cs:57]: Don't you have to do this for other platforms as well?

File: Trimble.Modus.Components/Controls/ListView/TMListView.xaml.cs [hunk 2]
```diff
@@ -51,8 +49,13 @@ public new IEnumerable ItemsSource
     public TMListView()
     {
         HasUnevenRows = true;
+        if (DeviceInfo.Platform == DevicePlatform.WinUI)
+        {
+            SetDynamicResource(BackgroundColorProperty, "CellDefaultBackgroundColor");
```
Comment (rthanga1) [inline Trimble.Modus.Components/Controls/ListView/TMListView.xaml.cs:54]: Not needed for other platforms?

**Review by rthanga1 — APPROVED** [review]

File: Trimble.Modus.Components/Controls/ListView/TMListView.xaml.cs (hunk 2 above)
Comment (Sofiya-kumar) [reply Trimble.Modus.Components/Controls/ListView/TMListView.xaml.cs:54]: Yes this for list view background color

File: Trimble.Modus.Components/Controls/ListView/TextCell.xaml.cs (hunk 1 above)
Comment (Sofiya-kumar) [reply Trimble.Modus.Components/Controls/ListView/TextCell.xaml.cs:57]: Yeah, the selected background color is not visible in windows when giving the background color. For windows, we are giving the colors in the App.xaml to support this we are giving the background as white in list view and not setting the background of view cell

**Pull request merged**